jmxexporter-prometheus-grafana/docker-compose.override.yml

```diff
@@ -79,7 +79,7 @@ services:
       - $MONITORING_STACK/assets/prometheus/security:/etc/kafka/secrets
 
   grafana:
-    image: grafana/grafana:7.0.6
+    image: grafana/grafana:7.3.7
```

Comment:
Any reason for not bumping it to the latest 7.4.3 version instead ?

Reply:
no particular reason, 7.3.7 was the latest at the time of starting my project. I tested with 7.4.3 and everything appears alright.

Reply:
Please bump it to 7.4.3